### ISSUE TITLE
instructions for running server

### ISSUE THREAD

**Issue by d-d-d-dalia** [learn-co-curriculum/omniauth_readme#33]
This readme should specify that after running thin start --ssl, to navigate to https:// + the address generated. It trips students up.

**Comment by SeeYouSpaceCowboy** [comment]
Yeah, def is a good idea to specify that. They default to just `localhost:3000` instead of `https://localhost:3000`
